client/src/app/app.component.scss

```diff
@@ -0,0 +1,37 @@
+// Standard Styles
+body {
+    /* Margin bottom by footer height */
+    margin-bottom: 60px;
+  }
+  .footer {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    /* Set the fixed height of the footer here */
+    height: 60px;
+    line-height: 60px; /* Vertically center the text there */
```

Comment:
This looked funny to me, but I read up about it on [this minimalist page](http://www.vanseodesign.com/blog/demo/vertical-centering/line-height.php) and I understand now that it pads the container above and below the content. The one caveat was that it must be only a single, unbroken line, so as long as we're diligent about that, we'll be good.